packages/pyright-internal/src/languageService/autoImporter.ts

```diff
@@ -279,11 +279,11 @@ export class AutoImporter {
 
             if (
                 !isStubOrHasInit.isStub &&
-                autoImportSymbol.kind === CompletionItemKind.Variable &&
-                /[a-z]/.test(name)
+                (autoImportSymbol.kind === undefined || autoImportSymbol.kind === CompletionItemKind.Variable) &&
+                !SymbolNameUtils.isPublicConstantOrTypeAlias(name)
```

Comment:
previously, name check was negative check based on the fact that the lexer has used proper identifier check (https://github.com/microsoft/pyright/blob/master/packages/pyright-internal/src/parser/characters.ts#L52) which allows non enlgish unicode as variable name.

now, only English name can be used in auto import variable. probably okay? but want to make sure that everyone is okay with that.

ex) before one could have 검은색상값 = "BLACK" and it will show up in auto import, now it won't.

Reply:
@heejaechang, good question. Are you familiar with the naming conventions used in locales that don't use Roman (ASCII) characters? Would it be common to see someone use "검은색상값" as the name of a type alias, or would that be unexpected and atypical? If it's expected, then is there a common heuristic we could use here? How about "starts with a capital letter and has no underscores?"

Reply:
I think using non-roman character as a variable name is not common (in Korea at least) since usually, tool support is so bad even if language spec allows it :) but that being said, I believe in some countries (one of them is Japan I believe), it is used more often.

but, this is only for auto-import, but not actual variable name, so might be okay?

Reply:
OK, sounds good to me. Let's stick with this for now. If it's too restrictive, I'm sure we'll get feedback.